sssd_test_framework/utils/override.py

```diff
@@ -0,0 +1,375 @@
+"Managing local users and groups."
+
+from __future__ import annotations
+
+import jc
+from pytest_mh import MultihostHost, MultihostUtility
+from pytest_mh.cli import CLIBuilder, CLIBuilderArgs
+from pytest_mh.ssh import SSHLog
+
+__all__ = [
+    "OverrideUtils",
+    "OverrideUser",
+    "OverrideGroup",
+]
+
+
+class OverrideUtils(MultihostUtility[MultihostHost]):
```

Comment:
It  would be better to rename it to LocalOverrideUtils (also use local overrides in comments).